src/main/java/com/show/showticketingservice/service/PerformanceService.java

```diff
@@ -157,4 +171,56 @@ private void checkCorrectPerfTime(PerformanceTimeRequest timeRequest) {
         }
 
     }
+
+    private List<SeatRequest> setSeatInfo(int performanceId, List<PerformanceTimeRequest> performanceTimeRequests) {
+        /*
+        좌석 정보 세팅
+        - DB로부터 좌석 정보에 필요한 정보를 불러옴 (VenueHallColumnSeat, SeatPriceRowNumData)
+         */
+
+        VenueHallColumnSeat venueHallColumnSeat = venueHallService.getVenueHallColumnAndId(performanceId);
+
+        List<SeatPriceRowNumData> seatPriceRowNumDataList = seatPriceService.getSeatPriceRowNum(performanceId);
+        List<SeatRequest> seatRequests = new ArrayList<>();
+
+        for(PerformanceTimeRequest performanceTimeRequest : performanceTimeRequests) {
```

Comment:
이런 것도 stream 연산으로 바꿔보시면 좋을 것 같네요~

```suggestion
        performanceTimeRequests.stream().forEach(performanceTimeRequest -> {
```